packages/stores/src/account/__tests_e2e__/create-pool.spec.ts

```diff
@@ -3,15 +3,18 @@ import {
   chainId,
   deepContained,
   getEventFromTx,
+  initAccount,
   RootStore,
   waitAccountLoaded,
 } from "../../__tests_e2e__/test-env";
 
 describe("Create Pool Tx", () => {
   const { accountStore } = new RootStore();
 
+  let account: ReturnType<(typeof accountStore)["getWallet"]>;
   beforeAll(async () => {
-    const account = accountStore.getWallet(chainId);
+    await initAccount(accountStore, chainId);
```

Comment:
Can we just have initAccount do everything given the params:
* Get the account
* Wait for it to be loaded
* Finally return a _defined_ account